tests/NgTests/Data/TestCatalogEntries.resx

```diff
@@ -172,6 +172,247 @@
   }
 }</value>
   </data>
+  <data name="TestCatalogPageForMyPackage" xml:space="preserve">
+    <value>{
+  "@id": "http://tempuri.org/page0.json",
+  "@type": "CatalogPage",
+  "commitId": "8a9e7694-73d4-4775-9b7a-20aa59b9773e",
+  "commitTimeStamp": "2017-02-08T17:18:55.3335317Z",
+  "count": 3,
+  "items": [
+    {
+      "@id": "http://tempuri.org/data/2017.02.08.17.16.18/mypackage.3.0.0.json",
+      "@type": "nuget:PackageDetails",
```

Comment:
nit: can we put this in non-chronological order? Really we should test all of the order variants (ABC - asc, CBA - desc, BAC - unordered) to make sure catalog items that are coincidentally in order do not hide a bug.